DTGB-910: Fix Sass deprecations

lennartvava opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

Commit:
STIJ-446: Fix Sass deprecations part 1

Commit:
STIJ-446: Fix js validate warnings; Set node version to 22

Commit:
DTGB-910: Fix npm deprecations where possible

Commit:
DTGB-910: Fix non breaking npm audit issues

Commit:
DTGB-910: Update changelog for version 7.0.0

Commit:
Merge branch '7.x' into feature/STIJ-446

Commit:
DTGB-910: Update travis config

Commit:
DTGB-910: Fix Sass deprecations part 2

Commit:
DTGB-910: Update starterkit and changelog

Commit:
DTGB-910: Update sass @use statements to optimize code

Commit:
DTGB-910: Update to styleguide 7.0.1 + optimize sass setup

zero2one commented on CHANGELOG.md:
```diff
@@ -2,6 +2,22 @@
+See more change instructions in the gent_styleguide 7.x changelog.
+
+### changed
+- DTGB-910: Update gent_styleguide to 7.0.0.
```
There should be an empty line after a heading in markdown.

zero2one commented on CHANGELOG.md:
```diff
@@ -2,6 +2,22 @@
 
 All Notable changes to `StadGent/drupal_theme_gent-base`.
 
+## [7.0.0]
```
There is no 7.0.0 release yet...

```markdown
## [Unreleased]
```

lennartvava replied:
This will be 7.0.0

zero2one replied:
Then it should be updated from [Unreleased] to [7.0.0] in the 7.0.0 release branch.

zero2one commented on source/sass/21-atoms/input/_input.scss:
```diff
@@ -0,0 +1,4 @@
+@forward 'autocomplete';
+@forward 'checkbox';
+@forward 'radio';
+@forward 'tel';
```
@use should be enough?

zero2one commented on source/sass/61-layouts/_layouts.scss:
```diff
@@ -0,0 +1,2 @@
+@forward 'detail-layout';
+@forward 'overview-layout';
```
@use is enough

zero2one commented on source/sass/modules/README.md:
```diff
@@ -0,0 +1,9 @@
+# MODULES DIRECTORY
+This directory is used to create easy-access sass resources pointing
```
Empty line after heading.

zero2one commented on source/gulpfile.js:
```diff
@@ -4,12 +4,43 @@ import gulp from 'gulp';
+    }))
+    .pipe(gulpif(build, sassLint.failOnError()))
+    .pipe(sassLint.format())
+    .pipe(sassLint.failOnError());
```
This line is not needed, it is a duplicate of line 39.

Commit:
DTGB-910: Update gulp validation tasks

Fix validation tasks to make sure the watch task is not unterrupted
when errors occur.

zero2one commented on source/gulpfile.js:
```diff
@@ -4,12 +4,43 @@ import gulp from 'gulp';
+    }))
+    .pipe(gulpif(build, sassLint.failOnError()))
+    .pipe(sassLint.format())
+    .pipe(gulpif(build, sassLint.failAfterError()));
```
This line is not needed, already tackled on line 39.
